tests/unit_tests/gebt_poc/test_solver.cpp

```diff
@@ -508,7 +475,50 @@ TEST(SolverTest, CalculateIterationMatrixComponents) {
     );
 }
 
-TEST(SolverTest, CalculateStaticIterationMatrixNonZeroValues) {
+TEST(SolverTest, CalculateStaticIterationMatrixWithZeroValues) {
+    // 5 nodes on a line
+    auto position_vectors = gen_alpha_solver::create_vector({
+        0., 0., 0., 1., 0., 0., 0.,  // node 1
+        1., 0., 0., 1., 0., 0., 0.,  // node 2
+        2., 0., 0., 1., 0., 0., 0.,  // node 3
+        3., 0., 0., 1., 0., 0., 0.,  // node 4
+        4., 0., 0., 1., 0., 0., 0.   // node 5
+    });
+
```

Comment:
I'm having trouble knowing when there should be 6 values per row (per node) vs when there should be 7 values per row. But that's probably not a significant issue, probably just me not keeping up with the math development lately.

Reply:
@alanw0 Yup, this is a direct consequence of the SO3-based mathematical development we are using. Please take a look at the following excerpt - basically position and displacements would contain 7 components/node, while velocity, acceleration will contain 6 components/node.

![image](https://github.com/Exawind/openturbine/assets/59463700/53dc9adc-47e8-4825-9532-31b942641ec5)

Reply:
Please feel free to ask more question - we should get one approval before we merge this PR